_includes/projectlist.html

```diff
@@ -2,8 +2,8 @@
   <h3>Projects currently in this phase:</h2>
   <ul>
   {% for project in site.data.projects %}
-    {% if project[1].stage == include.stage %}
-    <li><a href="{{site.baseurl}}/project/{{project[0]}}">{{ project[1].project }}</a></li>
+    {% if project[1]['stage'] == include.stage %}
```

Comment:
Well, it's currently broken using that syntax. https://18f.gsa.gov/dashboard/stages/#discovery Maybe I'm misunderstanding something.